WasatchNET/EEPROM.cs

```diff
@@ -38,6 +38,9 @@ public class EEPROM : IEEPROM
         /// </remarks>
         internal const int MAX_PAGES = 8;
         internal const int MAX_PAGES_REAL = 512;
+        internal const int MAX_NAME_LEN = 16;
+        internal const int MAX_LIB_ENTRIES = 8;
+        internal const int PIXELS = 1952;

```

Comment:
Wasatch.NET is used for every spectrometer we build.  We cannot hardcode "PIXELS = 1952" into a general-purpose spectrometer driver.  Perhaps LIBRARY_SIZE_PIXELS?

Reply:
> Are we saying that the maximum possible memory is 8 entries of 1952 pixels?

Yep.  1952px = 3904 bytes = 61 EEPROM pages
8 x 61 pages = 488 pages
The EEPROM only has 512 pages, so...

Honestly, 8 is a perfectly reasonable number.  SiG-LP is explicitly tasks with *authenticating* against a single compound (not "identifying" across a broad library), so we actually expected to only need one library spectrum in the EEPROM, period.  That's why the original feature implementation only supported a single library spectrum.

However, testing determined that the library matching works "better" if it has multiple compounds to compare against.  It's kind of like we're asking it to authenticate if a picture is a dog or not.  With nothing but a picture of one dog, it can "kind" of do that, but if you show it a cat, it sees quite a number of dog-like features (quadruped mammal).  If we can give the algorithm a few "non-dog" pictures of cats, rabbits, spiders etc, it does a much better job of matching dogs.

As they say, eight is enough...